helm_deploy/use-of-force/templates/ingress_v1_2.yaml

```diff
@@ -0,0 +1,32 @@
+{{- if and .Values.ingress.enabled .Values.ingress.v1_2_enabled -}}
+{{- $fullName := include "app.fullname" . -}}
+{{- $ingressName := print $fullName "-v1-2" }}
+{{- $ingressPath := .Values.ingress.path -}}
+apiVersion: networking.k8s.io/v1
+kind: Ingress
+metadata:
+  name: {{ $ingressName }}
+  labels:
+    {{- include "app.labels" . | nindent 4 }}
+  annotations:
+    nginx.ingress.kubernetes.io/custom-http-errors: "418"
+    external-dns.alpha.kubernetes.io/set-identifier: {{ $ingressName }}-{{ .Release.Namespace }}-green
```

Comment:
This is what the generic-serivce is doing (seems slightly different to what we we're doing before). Though I thought it'd be better to align with the generic-service now whilst we're moving Ingress anyways.

Oddly when templated, it's showing up as **use-of-force-prod-v1-2-default-green**. Assuming this is because it's not running in the context of the CircleCI **deploy_env** job.

Thought it was worth mentioning in case anyone has better foresight than I do. I suppose we'll see when it gets deployed to dev if the names have stayed the same 👀

Reply:
wonder if it might be because we have a default namespace set in the kubeconfig context and helm reads it from there? 
https://github.com/ministryofjustice/hmpps-circleci-orb/blob/e2abfc77c9bb08bdf8c96cf9e57c841f131b052f/src/commands/k8s_setup.yml#L33-L36